frontend/apps/reader/modules/readergesture.lua

```diff
@@ -97,6 +97,7 @@ function ReaderGesture:buildMenu(ges, default)
         {_("Reading progress"), "reading_progress", ReaderGesture.getReaderProgress ~= nil},
         {_("Full screen refresh"), "full_refresh", true},
         {_("Night mode"), "night_mode", true},
+        {_("Open previous document"), "open_previous_document", true},
```

Comment:
This list is getting quite crowded :)
I would move this Open previous document nearest to History (same kind of category for me).
It may need to be sorted. I'd see various categories (with separators if possible):
- document page navigation
- document info panels (toc, bookmarks, reading progress).
- other documents list or switch (history, Open previous)
- device stuff (night mode, suspend, frontlight...)